spec/codecs/es_bulk_spec.rb

```diff
@@ -5,7 +5,7 @@
 require 'logstash/plugin_mixins/ecs_compatibility_support/spec_helper'
 
 describe LogStash::Codecs::ESBulk, :ecs_compatibility_support  do
-  ecs_compatibility_matrix(:disabled, :v1, :v8) do |ecs_select|
+  ecs_compatibility_matrix(:disabled, :v1, :v8 => :v1) do |ecs_select|
```

Comment:
[here](https://github.com/logstash-plugins/logstash-codec-es_bulk/pull/20#discussion_r673056749) pointed out that we want to test with the setting v8.
I believe both ways can test v8, and the mapping gives handy `ecs_select`. 
Do you think testing with `(:disabled, :v1, :v8)` is more aligned to the test purpose? 
(I think my original test which exclude :disabled is wrong anyway)